### PR TITLE
Improve admin panel with more information 

### PR DESCRIPTION
**Pull request by ymasiuk**

### CODE REVIEW

File: app/views/admin/trades/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<h3><%= t('.trades') %></h3>
+<div class="card">
+  <%= table_for @trades, class: 'table table-bordered table-striped table-condensed' do |t| %>
+    <% t.column :id, "ID" %>
```
Comment (gfedorenko) [inline app/views/admin/trades/index.html.erb]: Please, put single quotes for strings in erb tags

File: config/templates/application.yml.erb [hunk 2]
```diff
@@ -164,6 +164,7 @@ defaults: &defaults
   RANGER_HOST: '0.0.0.0'
   RANGER_PORT: '8081'
 
+  COPYRIGHT_TITLE: Peatio.tech
```
Comment (ysv) [inline config/templates/application.yml.erb:168]: Add description for this env

File: app/controllers/admin/members_controller.rb [hunk 3]
```diff
@@ -6,9 +6,10 @@ class MembersController < BaseController
     load_and_authorize_resource
 
     def index
+      @q = Member.ransack(params[:q])
       @search_field = params[:search_field]
       @search_term  = params[:search_term]
```
Comment (ysv) [inline app/controllers/admin/members_controller.rb:11]: do we need class variables here ?
may be
`search_field` and `search_term`

**Review by ysv — CHANGES_REQUESTED** [review]

File: config/templates/application.yml.erb (hunk 2 above)
Comment (chumaknadya) [reply config/templates/application.yml.erb:168]: Done

File: app/controllers/admin/members_controller.rb (hunk 3 above)
Comment (gfedorenko) [reply app/controllers/admin/members_controller.rb:11]: I think, it was used on the view

**Pull request closed without merging**